schemas/dublin-core/src/main/plugin/dublin-core/loc/fre/labels.xml

```diff
@@ -141,7 +141,7 @@
   </element>
 
   <element name="dc:coverage">
-    <label>Etendue géographique</label>
+    <label>Étendue géographique</label>
```

Comment:
Not sure why characters are not displayed properly on github. File looks fine with UTF-8 encoding in source.